Examples/Scripts/Python/truth_tracking.py

```diff
@@ -14,6 +14,7 @@ def runTruthTracking(
     outputDir: Path,
     digiConfigFile: Path,
     directNavigation=False,
+    reverseFilteringMomThreshold=0*u.GeV,
```

Comment:
This effectively switches reverse filtering off, right?

Reply:
Exactly, which is what the previous default was.